ui/src/app/base/selectors/auth/auth.test.js

```diff
@@ -1,72 +1,69 @@
 import auth from "./auth";
+import { userState } from "testing/factories";
```

Comment:
I think the convention `import { userState as userStateFactory }` might be worth adopting (I'm doing that in my branch), to make it clear what these functions are.